Documentation/kubernetes.md

```diff
@@ -46,83 +44,44 @@ osd-6jmph                   1/1       Running   0          1m
+- **[Shared File System](k8s-filesystem.md)**: Create a file system to be shared across multiple pods
+- **[Object](k8s-object.md)**: Create an object store that is accessible inside or outside the Kubernetes cluster
 
-```
-$ kubectl get pvc
-NAME             STATUS    VOLUME                                     CAPACITY   ACCESSMODES   AGE
-mysql-pv-claim   Bound     pvc-95402dbc-efc0-11e6-bc9a-0cc47a3459ee   20Gi       RWO           1m
-wp-pv-claim      Bound     pvc-39e43169-efc1-11e6-bc9a-0cc47a3459ee   20Gi       RWO           1m
-```
-
-Once the wordpress and mysql pods are in the `Running` state, get the cluster IP of the wordpress app and enter it in your brower:
-
-```
-$ kubectl get svc wordpress
-NAME        CLUSTER-IP   EXTERNAL-IP   PORT(S)        AGE
-wordpress   10.3.0.155   <pending>     80:30841/TCP   2m
-```
-
-You should see the wordpress app running.  
-
-**NOTE:** When running in a vagrant environment, there will be no external IP address to reach wordpress with.  You will only be able to reach wordpress via the `CLUSTER-IP` from inside the Kubernetes cluster.
+## Tools
 
 ### Rook Client
 You also have the option to use the `rook` client tool directly by running it in a pod that can be started in the cluster with:
-```
+```bash
 kubectl create -f rook-client/rook-client.yml
-```  
 
-Starting the rook-client pod will take a bit of time to download the container, so you can check to see when it's ready with (it should be in the `Running` state):
-```
+# Starting the rook-client pod will take a bit of time to download the container, so check when it's in the Running state
 kubectl -n rook get pod rook-client
-```
 
-Connect to the rook-client pod and verify the `rook` client can talk to the cluster:
-```
+# Connect to the rook-client pod 
 kubectl -n rook exec rook-client -it bash
+
+# Verify the rook client can talk to the cluster:
 rook node ls
 ```
 
 At this point, you can use the `rook` tool along with some [simple steps to create and manage block, file and object storage](client.md).
 
+### Advanced Configuration and Troubleshooting
+We have created a toolbox container that contains the full suite of Ceph clients for debugging and troubleshooting your Rook cluster.  Please see the [toolbox readme](toolbox.md) for setup and usage information.
+
 ### Monitoring
 Each Rook cluster has some built in metrics collectors/exporters for monitoring with [Prometheus](https://prometheus.io/).
 To learn how to set up monitoring for your Rook cluster, you can follow the steps in the [monitoring guide](./k8s-monitoring.md).
 
-### Teardown
-To clean up all the artifacts created by the demo, run the following:
-```
-kubectl delete -f wordpress.yaml
-kubectl delete -f mysql.yaml
+## Teardown
+To clean up all the artifacts created by the demo, *first cleanup the resources from the block, file, and object walkthroughs* (unmount volumes, delete volume claims, etc), then run the following:
```

Comment:
maybe bold instead of (or in addition to italics) here because it's an important step that shouldn't be easy to skip?